### PR TITLE
[docs] terraform-modules automation

### PR DESCRIPTION
**Pull request by alebabai**
## what
* bump build-harness tag to 0.14.5
* create terraform-module-page template for docs project
* create script for automation docs create/update actions

## why
https://github.com/cloudposse/docs/issues/228
https://github.com/cloudposse/docs/issues/229

## Expected result
Script will create terraform-pages in the `content/terraform-modules/` folder

### CODE REVIEW

**Comment by osterman** [comment]
We need documentation to be self-contained. Putting this in the `dev` folder is the wrong place and causes unnecessary sprawl.

The scripts @vadim-hleif have in this repo were more for hacking together pull-requests, but not for a routine workflow. 

The `terraform-module-page` looks good. Let's move this into the `docs` project under a `templates/` root folder.

File: scripts/docs-update-terraform-modules.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+module_template=templates/terraform-module-page.md
+
+for module_dir in projects/terraform/*
+do
+    module=$(basename $module_dir)
+    module_readme_yaml=$module_dir/readme.yaml
+    module_page=projects/docs/content/terraform-modules/$module.md
+
+    if [ -f $module_readme ]; then
+        make readme README_TEMPLATE_FILE=$module_template README_FILE=$module_page README_YAML=$module_readme_yaml README_INCLUDES=$(pwd)/$module_dir
```
Comment (osterman) [inline scripts/docs-update-terraform-modules.sh:13]: Move this script into the docs repo. Perhaps under a `scripts/` folder

**Comment by alebabai** [comment]
@osterman 
> Putting this in the dev folder is the wrong place and causes unnecessary sprawl.

i've used dev repo because it already contains all terraform-modules repositories and docs repository in one place. So it is very suitable place for orchestration things in my point of view.
Moving this scripts to docs repo will require fetching names and cloning all terraform-modules (and coping code for it)

**Comment by alebabai** [comment]
@osterman i'll try to implement it in docs repo

**Comment by alebabai** [comment]
replaced by https://github.com/cloudposse/docs/pull/247

**Pull request closed without merging**